config/src/main/java/com/redhat/lightblue/config/LightblueFactory.java

```diff
@@ -345,15 +346,15 @@ public Factory getFactory()
         return factory;
     }
 
-    public Mediator getMediator()
+    public Mediator getMediator(RequestMetrics metrics)
```

Comment:
Altered this approach to pass metricCtx to all mediator commands, for better metric tracking.